melodic/distribution.yaml

```diff
@@ -9439,6 +9439,11 @@ repositories:
       url: https://github.com/UTNuclearRoboticsPublic/rosthrottle-release.git
       version: 1.2.0-3
     status: maintained
+  roswasm_suite:
+    doc:
+      type: git
+      url: https://github.com/nilsbore/roswasm_suite.git
```

Comment:
Looking at the source repository, it seems like https://github.com/nilsbore/roswasm_suite/blob/master/roscpp_json_serialize/package.xml (at least) still has a TODO for the license.  Can you update that please?